nautobot/docs/configuration/required-settings.md

```diff
@@ -149,25 +149,6 @@ setup](https://github.com/Suor/django-cacheops#setup).
 !!! warning
     [`CACHEOPS_REDIS`](#cacheops_redis) and [`CACHEOPS_SENTINEL`](#cacheops_sentinel) are mutually exclusive and will result in an error if both are set.
```

Comment:
I see there's a link under this `CACHEOPS_SENTINEL` section to the `Caching` guide, but there's no such link under the above `CACHEOPS_REDIS` section. Can you take a stab at adding some verbiage there?